portal/server/routes/api-data-provider.js

```diff
@@ -84,12 +84,12 @@ const provide = (listOfDataTypes) => async (req, res, next) => {
     req.apiData = {};
   }
 
-  const promises = [];
-  listOfDataTypes.forEach(async (dataType) => {
-    promises.push(get(dataType, req));
-  });
+  Promise.all(
+    listOfDataTypes.map(async (dataType) => {
+      return get(dataType, req);
+    }),
```

Comment:
This can just be `listOfDataTypes.map(async (dataType) => await get(dataType, req))`